addon/webextension/README.md

```diff
@@ -8,7 +8,7 @@ This contains all the files for the WebExtension.  Most files are not "built", b
   2. The selector content worker, in `selector/`
   3. The site helper that communicates with the Page Shot website, in `site-helper.js`
 - The background process is loaded according to a list in `manifest.json.template` – if there are load order dependencies, they must be manually managed with that list
-- The selector content worker is loaded with a list in `background/loadSelector.js`.  Again this must be manually maintained.
+- The selector content worker is loaded with a list in `background/selectorLoader.js`.  Again this must be manually maintained.
```

Comment:
I combined the functionality of `loadSelector.js` and `lifecycle.js` into `selectorLoader.js`. It was a bit easier for me to reason through it that way.